locales/de.json

```diff
@@ -49,25 +79,275 @@
 	"administration": {
 		"button": {
 			"changePassword": "Passwort ändern",
+			"classInSchoolYear": "Klasse in Schuljahr",
 			"consent": "Einwilligungen:",
 			"deleteUser": "Benutzer löschen",
 			"edit": "Bearbeiten",
 			"generatePersonalInvitationLink": "Persönlichen Einladungslink generieren",
 			"printListWithAccessData": "Liste mit Zugangsdaten drucken",
 			"receivedConsentOnPaper": "Einverständnis auf Papier erhalten",
-			"sendTheInvitationLinkByMail": "Einladungslink per E-Mail versenden"
+			"sendTheInvitationLinkByMail": "Einladungslink per E-Mail versenden",
+			"transfer": "versetzen",
+			"transferClassToTheNextSchoolYear": "Klasse ins nächste Schuljahr versetzen",
+			"verify": "Verifizieren"
+		},
+		"classes": {
+			"button": {
+				"addClass": "Klasse hinzufügen",
+				"addStudentsFromPreviousYearClass": "Schüler aus Vorjahresklasse hinzufügen",
+				"copyLink": "Link kopieren",
+				"declareConsentOnBehalfOfTheStudents": "Einverständnis im Namen der Schüler erklären",
+				"printLinkWithQRCode": "Link mit QR-Code Drucken"
+			},
+			"label": {
+				"addStudentFromPreviousYearClass": "Schüler aus Vorjahres-Klasse hinzufügen",
```

Comment:
avoid similar translations with different spelling, you could reuse or type in the same manner `Schüler aus Vorjahresklasse hinzufügen`